akka-actor-tests/src/test/scala/akka/dispatch/VirtualThreadDispatcherSpec.scala

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright (C) 2009-2025 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.dispatch
+
+import akka.actor.ActorSystem
+import akka.testkit.TestKit
+import akka.testkit.TestProbe
+import akka.util.JavaVersion
+import com.typesafe.config.ConfigFactory
+import org.scalatest.wordspec.AnyWordSpec
+
+class VirtualThreadDispatcherSpec extends AnyWordSpec {
+
+  "The virtual thread support" should {
+
+    "run tasks on virtual threads" in {
+      if (JavaVersion.majorVersion < 21) {
+        // loom not available yet here
+        pending
+      } else {
+        var system: ActorSystem = null
```

Comment:
`val system` and create it here. if creation fails it doesn't have to be shut down